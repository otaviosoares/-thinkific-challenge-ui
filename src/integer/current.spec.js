import React from 'react';
import { shallow } from 'enzyme';
import Current from './current';

test('it renders', () => {
  var component = shallow(
    <Current current="10" />
  )
  expect(component).toMatchSnapshot();
})