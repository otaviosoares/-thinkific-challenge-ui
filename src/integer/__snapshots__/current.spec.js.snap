// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<div
  className="bg-info text-center current"
>
  Current integer is:
  <br />
  <strong>
    10
  </strong>
</div>
`;
